src/System.Collections/tests/Resources/System.Collections.Tests.rd.xml

```diff
@@ -1,7 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
-  <Library>
+  <Library Name="System.Collections.Tests">
+    <Assembly Name="*Application*" DoNotInline="true" DoNotOptimize="true" />
```

Comment:
Thanks for taking care of it. Please do add a comment mentioning VSTS bug number 472947.

@MichalStrehovsky don't worry about cleaning this things up, I've been submitting the PRs and porting them to clean up all the tests projects that have ILCBuildType=chk and will do so for this one as well whenever the bug is fixed. I think this is the last test project with ILCBuildType=chk.